crates/indexer/src/processors/default_processor.rs

```diff
@@ -151,7 +151,20 @@ fn insert_user_transactions_w_sigs(
             diesel::insert_into(schema::user_transactions::table)
                 .values(&all_user_transactions[start_ind..end_ind])
                 .on_conflict(ut_schema::version)
-                .do_nothing(),
+                .do_update()
+                .set((
+                    ut_schema::block_height.eq(excluded(ut_schema::block_height)),
+                    ut_schema::parent_signature_type.eq(excluded(ut_schema::parent_signature_type)),
+                    ut_schema::sender.eq(excluded(ut_schema::sender)),
+                    ut_schema::sequence_number.eq(excluded(ut_schema::sequence_number)),
+                    ut_schema::max_gas_amount.eq(excluded(ut_schema::max_gas_amount)),
+                    ut_schema::expiration_timestamp_secs
+                        .eq(excluded(ut_schema::expiration_timestamp_secs)),
+                    ut_schema::gas_unit_price.eq(excluded(ut_schema::gas_unit_price)),
+                    ut_schema::timestamp.eq(excluded(ut_schema::timestamp)),
+                    ut_schema::entry_function_id_str.eq(excluded(ut_schema::entry_function_id_str)),
+                    ut_schema::inserted_at.eq(excluded(ut_schema::inserted_at)),
+                )),
```

Comment:
There's no explanation for this change?

Reply:
So whether we override the results or not doesn't matter. I first had it as upsert, then changed it to not replace, but now we do want it to replace b/c we want to backfill some data. We can change it tomorrow back to not replace and it wouldn't matter. For these tables our processing should be idempotent.

Reply:
Let's in the future keep the commits tied to the changes you're making, and add `[indexer]` to the beginning of to commit messages.  I'm not worried about the change you're making, it's just that it gets lost in the history when you see "fix max timestamp" and then all of a sudden it's doing something else.